File: two_sum/two_sum.php
<?php

class Solution
{
    public function twoSum($nums, $target): array
    {
        for ($i = 0; $i < count($nums) - 1; $i++) {
            for ($j = $i + 1; $j < count($nums); $j++) {
                if ($nums[$i] + $nums[$j] === $target) {
                    return [$i, $j];
                }
            }
        }

        return [];
    }
}

$s = new Solution();

$nums = [2,7,11,15];
$target = 9;

$r = $s->twoSum($nums, $target);

var_dump($r);

File: two_sum/two_sum.py
from typing import List, Tuple

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i, _ in enumerate(nums):
            for j in range(i+1, len(nums)):
                if nums[i] + nums[j] == target:
                    return [i, j]

        return []

nums = [2,7,11,15]
target = 9

s = Solution()
r = s.twoSum(nums, target)

print(r)
